### ISSUE TITLE
[[72] Into the Fold (11978)] - Doesn't Show, but it's Available

### ISSUE THREAD

**Issue by Gogo1951** [Questie/Questie#4499]
https://github.com/Questie/Questie/issues/3902 - marked as fixed here, but still and issue as of today on latest.

**Comment by Gogo1951** [comment]
By the way, huge improvement on Borean Tundra and Grizzly Hills overall.

Just few stragglers still.

**Comment by Yttrium-tYcLief** [comment]
Can you confirm you have not completed `11977` *nor* `11979`?

**Comment by Gogo1951** [comment]
No, just smashing out quests. Sorry. No way to check now but it wasn't working when I went through.

/script print(C_QuestLog.IsQuestFlaggedCompleted(11979))
/script print(C_QuestLog.IsQuestFlaggedCompleted(11977))

Both returned false.

**Comment by Yttrium-tYcLief** [comment]
Hmm... from everything I could tell you definitely had to access it from one of those pre-reqs. Guess I'll dig into this further.
